### ISSUE TITLE
incompatible with Mongoid version `v7.0`

### ISSUE THREAD

**Issue by wuhuizuo** [mongoid/mongoid-locker#51]
mongoid has release a new version: `v7.0` at 2018-3-12

but there have not a hight version limit for `mongoid-lock`, and only support `v2` to `v6` in lib/mongoid/locker/wrapper.rb

```text
RuntimeError:
  incompatible Mongoid version
# /usr/local/lib/ruby/gems/2.4.0/gems/mongoid-locker-0.3.5/lib/mongoid/locker/wrapper.rb:14:in `<top (required)>'
```

**Comment by dblock** [comment]
Fixed in #52

**Comment by dblock** [comment]
Closed via #52
